Compile error after changing the definition of `struct _static_thread_data`

zephyr/include/zephyr/kernel.h:712:9: error: designator order for field '_static_thread_data::init_name' does not match declaration order in '_static_thread_data'

after https://github.com/zephyrproject-rtos/zephyr/commit/41e8b44619c7dd60ccf91ac642c49e68d14f7ede

--- a/include/zephyr/kernel.h
+++ b/include/zephyr/kernel.h
@@ -707,8 +707,8 @@ struct _static_thread_data {
 	.init_p3 = (void *)p3,                                   \
 	.init_prio = (prio),                                     \
 	.init_options = (options),                               \
-	.init_delay = SYS_TIMEOUT_MS(delay),			 \
 	.init_name = STRINGIFY(tname),                           \
+	.init_delay = SYS_TIMEOUT_MS(delay),			 \
 	}
 
 /*

